File: src/main/java/com/zy/Hello.java
package com.zy;

public class Hello {
    public static void main(String[] args) {
        Person p = new Person("Tim",66.5f,1.7f);
        p.hello();
        p.Information();
        p.height = 1.7f;
        p.weight = 66.5f;
        System.out.println("BMI= "+p.bmi());
        /*int age = 19;
        Integer age2 = 19;
        char c = 'm';
        Character c2 = 'A';
        byte b = 120;
        float f = 120;
        float weight = 66.5f;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;
        String name = "Tom";*/
    }
}

File: src/main/kotlin/com/zy/Hello.kt
package com.zy

fun main() {
    //print("Hello Kotlin")
    var s :String? = "abcde"
    //s = null
    println(s?.length)
    println(s?.get(3))

    val h = Human("Tim",66.5f,1.7f)
    println(h.bmi())
    var score = 80
    println(score > 60)
    var c : Char = 'A'
    println(c.toInt() > 60)
   /* var age=19
    age=20
    var weight = 66.5
    var name : String = "Tang"
    print(name)*/
}

class Human(var weight : Float, var height : Float){

    init {
        println("test $weight")
    }

    constructor(name : String, weight: Float,height: Float) : this(weight,height){
        println("test2 $name")
    }



    fun bmi():Float{
        val bmi = weight / (height*height)
        return bmi
    }

    fun hello(){
        println("Hello Kotlin")
    }
}